src/main/java/nextstep/courses/domain/repository/CoverImageRepository.java

```diff
@@ -0,0 +1,12 @@
+package nextstep.courses.domain.repository;
+
+import nextstep.courses.domain.image.CoverImage;
+
+import java.util.Optional;
+
+public interface CoverImageRepository {
+    int save(CoverImage image, Long sessionId);
+
+    Optional<CoverImage> findBySessionId(Long id);
```

Comment:
```suggestion
    Optional<CoverImage> findBySessionId(Long sessionId);
```
파라메터명도 좀 더 명시적이면 좋을 것 같아요 :)